### PR TITLE
unify command descriptions

### PR DESCRIPTION
**Pull request by dsokal**
# Checklist

- [ ] I've added an entry to [CHANGELOG.md](https://github.com/expo/eas-cli/blob/main/CHANGELOG.md#main) if necessary. You can comment this pull request with `/changelog-entry [breaking-change|new-feature|bug-fix|chore] [message]` and CHANGELOG.md will be updated automatically.

# Why

We use a few different styles for command descriptions.

# How

Unify command descriptions.

# Test Plan

Run
- `echo "account\nbranch\nbuild\nchannel\ndevice\nproject\nsecret\nupdate\nwebhook" | xargs -I {} easd {} --help`
- `easd --help`
and verified all the command descriptions are written in the same style (except for the description for `eas help` but it'd require modifying `@oclif/plugin-help` .

### CODE REVIEW

File: packages/eas-cli/src/commands/secret/delete.ts [hunk 1]
```diff
@@ -18,8 +17,7 @@ import {
 import { promptAsync, toggleConfirmAsync } from '../../prompts';
 
 export default class EnvironmentSecretDelete extends EasCommand {
-  static description = `Delete an environment secret by ID.
-Unsure where to find the secret's ID? Run ${chalk.bold('eas secret:list')}`;
+  static description = `delete an environment secret by ID`;
```
Comment (jkhales) [inline packages/eas-cli/src/commands/secret/delete.ts:20]:
```suggestion
  static description = 'delete an environment secret by ID';
```

File: packages/eas-cli/src/commands/submit.ts [hunk 2]
```diff
@@ -41,8 +41,7 @@ interface CommandFlags {
 }
 
 export default class Submit extends EasCommand {
-  static description = `Submit build archive to App Store Connect
-See how to configure submits with eas.json: ${link('https://docs.expo.dev/submit/eas-json/')}`;
+  static description = `submit app binary to app store`;
```
Comment (jkhales) [inline packages/eas-cli/src/commands/submit.ts:44]:
```suggestion
  static description = 'submit app binary to app store';
```

**Review by jkhales — APPROVED** [review]
Thanks for doing this! 🙏

**Review by kbrandwijk — APPROVED** [review]
Looks great, the only thing I'm not sold on is the seemingly random use of 'on the current project' vs 'on this project' vs nothing, for example:

'delete a branch on the current project'
'list all branches on this project'
'rename a branch'

**Review by wkozyra95 — APPROVED** [review]

**Review by kbrandwijk — APPROVED** [review]

**Comment by dsokal** [comment]
/changelog-entry chore Unify EAS command descriptions style.

**Pull request merged**